consensus/polybft/polybft.go

```diff
@@ -108,6 +110,9 @@ type Polybft struct {
 
 	// tx pool as interface
 	txPool txPoolInterface
+
+	// relayer instance
+	relayer *relayer.Relayer
```

Comment:
Ideally, the relayer is not part of `polybft` in the future.

Reply:
Yeah, it's enough separated to just move it to one subcommand or a separate binary, but currently we're going to start it on `IsBridgeEnabled` inside polybft.